app/controllers/qsets_controller.rb

```diff
@@ -13,30 +13,31 @@ def index
   # handles the request to show all questions in a qset
   def show
     @feedback_active = !!current_user
-    # sorts by default by net votes; secondary sort by create date
-    @questions = Question.includes(:answers).where(qset_id: @qset.id).plusminus_tally.order(created_at: :desc)
-    if can? :see_all_questions, @qset
-      @filter_mine = true if cookies[:all_mine_other_filter] == 'mine'
-      @filter_other = true if cookies[:all_mine_other_filter] == 'other'
-      @filter_all = true unless @filter_mine or @filter_other
-    else
-      # show only the current user's questions
-      cookies[:all_mine_other_filter] = 'mine'
-      @filter_mine = true
-      @questions = @questions.where(user_id: current_user)
-    end
     @qsets = @qset.children
-    # a hash of qset question counts keyed by qset id
-    @subset_question_counts = @qsets.map do |s|
-      count = 0
-      s.self_and_descendants.each do |q|
-        scope = q.questions
-        scope = scope.where(user_id: current_user) if cannot? :see_all_questions, q
-        count += scope.count
+      # a hash of qset question counts keyed by qset id
+      @subset_question_counts = @qsets.map do |s|
+        count = 0
+        s.self_and_descendants.each do |q|
+          scope = q.questions
+          scope = scope.where(user_id: current_user) if cannot? :see_all_questions, q
+          count += scope.count
+        end
+        [s.id, count]
+      end.to_h
+    if @qset.settings(:permissions).toggle_qsets == 'questions'
+      # sorts by default by net votes; secondary sort by create date
+      @questions = Question.includes(:answers).where(qset_id: @qset.id).plusminus_tally.order(created_at: :desc)
+      if can? :see_all_questions, @qset
+        @filter_mine = true if cookies[:all_mine_other_filter] == 'mine'
+        @filter_other = true if cookies[:all_mine_other_filter] == 'other'
+        @filter_all = true unless @filter_mine or @filter_other
+      else
+        # show only the current user's questions
+        cookies[:all_mine_other_filter] = 'mine'
+        @filter_mine = true
+        @questions = @questions.where(user_id: current_user)
       end
-      [s.id, count]
-    end.to_h
-    if @qset.root?
+    elsif @qset.settings(:permissions).toggle_qsets == 'qset'
       render :organizationpage
     end
```

Comment:
For argument's sake, let's say one of the qsets had a value that wasn't `'qset'` or `'questions'`; it appears that they'd then be directed to the `show.html.erb` view, but would not have all the variable setup because they wouldn't hit the code under the first `if` block:
```
if @qset.settings(:permissions).toggle_qsets == 'questions'
  ...
```
Perhaps modifying the code so that the default (`'questions'`) is properly shown for an unknown setting is a wise thing to do; it is a bit more future-proof/safer. In other words, if I change the setting (through the console) to e.g. `flyinglizards` or some such nonsense, the app should probably handle that gracefully if possible (in my mind that means showing the `'questions'` view, since it's the default).